lib/vfscore/Config.uk

```diff
@@ -104,6 +104,10 @@ if LIBVFSCORE_AUTOMOUNT_CI
 		menu "Entry options"
 		config LIBVFSCORE_AUTOMOUNT_CI0_UKOPTS_MKMP
 		bool "mkmp - Create mountpoint"
+		config LIBVFSCORE_AUTOMOUNT_CI0_UKOPTS_IFINITRD0
```

Comment:
I think `KConfig` styling involves some indentation still but, alas, no config here seems to really respect that anyway so ok..